jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +132,207 @@ async def coroutine():
+            self.app.stop_progressing()
+
+        if error:
+            self.app.show_message(_(common_strings.error), error, tobefocused=self.menu_container)
+            return
+
+        if response is None:
+            self.app.show_message(_(common_strings.error), _("Error while creating client from SSA: empty server response"), tobefocused=self.menu_container)
+            return
+
+        if response.status_code not in (200, 201):
+            self.app.show_message(_(common_strings.error), _("Server {} returned error.\nStatus code: {}.\nResponse text: {}").format(iss, response.status_code, response.text), tobefocused=self.menu_container)
```

Ruff (0.15.11)

[error] Undefined name `iss`

(F821)